packages/styling/src/interfaces/IPalette.ts

```diff
@@ -242,4 +242,21 @@ export interface IPalette {
    * Color code for greenLight.
    */
   greenLight: string;
+
+  /**
+   * Color code for contrast black disabled.
+   */
+  contrastBlackDisabled: string;
```

Comment:
Missing newlines here and below between props

Reply:
Fixed this, but the comment wasn't marked as outdated.